src/FixedDataTable.js

```diff
@@ -959,7 +999,8 @@ var FixedDataTable = createReactClass({
     }
 
     var lastScrollToRow  = oldState ? oldState.scrollToRow : undefined;
-    if (props.scrollToRow != null && (props.scrollToRow !== lastScrollToRow || viewportHeight !== oldViewportHeight)) {
+    var rowHeightChanged = props.scrollToRow != null ? this._scrollHelper.getRowHeightChanged(props.scrollToRow) : 0
```

Comment:
Ah, I see what you're talking about.  This is an instance of issue #25 and #9, which basically point to the fact that FDT doesn't do dynamic row heights well.  

Rather than scroll to a row here, what we want to do is just trigger an update to our row heights and ensure we're not scrolled beyond the end.  Unfortunately the code currently does the row heights update separately in the render method of FixedDataTableBufferedRows and the scroll helper.  We're working to merge these to fix the bugs, but I think you'll need to wait for that work until we fix this without breaking existing behavior.  The latest of this work is in PR with #192.

Could you revert your latest change and then delete getRowHeightChanged from FixedDataTableScrollHelper (it should be unused now)?